### PR TITLE
Navigation Styling

### PR DESCRIPTION
**Pull request by DIWAKARKASHYAP**
## Link to issue

Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] Followed the repository's [Contributing Guidelines](/CONTRIBUTING.md).
- [x] I ran the app and tested it locally to verify that it works as expected.
- [ ] I have checked my code with an automatic accessibility tool such as Axe Dev Tools or Wave 
      and it shows no errors.
## Additional Information (Optional)
Any additional information that you want to give us.

### CODE REVIEW

File: components/Nav/NavPrimary.module.css [hunk 1]
```diff
@@ -19,26 +19,44 @@
...
-  display: block;
+  padding: 4px;
+  display: inline-block;
+  border-radius: 5px;
+  
 }
 
 .navListItem a:hover {
-  background-color: var(--primaryLt);
+  background-color: #28968C;
```
Comment (EmmaDawsonDev) [inline components/Nav/NavPrimary.module.css:36]: Please use the variable colors listed in the global css files.

File: components/Nav/NavPrimary.module.css [hunk 2]
```diff
@@ -19,26 +19,44 @@
...
 }
 
 .navList a:focus {
-  background-color: var(--primaryLt);
-  font-weight: bold;
+  background-color: #28968C;
```
Comment (EmmaDawsonDev) [inline components/Nav/NavPrimary.module.css:40]: Please use the variable colors listed in the global css file

File: components/Nav/NavPrimary.module.css [hunk 3]
```diff
@@ -19,26 +19,44 @@
...
+}
+
+.searchBox{
```
Comment (EmmaDawsonDev) [inline components/Nav/NavPrimary.module.css:43]: Please remove this from the PR

File: components/Nav/NavPrimary.module.css [hunk 4]
```diff
@@ -19,26 +19,44 @@
...
+}
+.searchBox:focus {
+  outline-color: var(--primaryLt);
+
 }
 
 @media (min-width: 40rem) {
   .navPrimary {
-    width: 15rem;
+    width: 12rem;
```
Comment (EmmaDawsonDev) [inline components/Nav/NavPrimary.module.css:58]: Please leave the width as it was for now

File: components/Nav/NavPrimary.tsx [hunk 5]
```diff
@@ -1,22 +1,36 @@
...
 import { pages } from "../../data/pages"
 import styles from "./NavPrimary.module.css"
+import { useState } from "react"
 
 export interface INavProps {
 	activeNavLink: string
 }
 
 export const NavPrimary = ({ activeNavLink }: INavProps) => {
+	const [search, setSearch] = useState<string>("")
```
Comment (EmmaDawsonDev) [inline components/Nav/NavPrimary.tsx:11]: Please remove all of the code to do with search

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Hi @DIWAKARKASHYAP thank you for your keenness to contribute to this project. We normally do not accept PRs that do not have a linked issue (Rule 1 of the contributor guidelines). However, I understand that sometimes it's hard to follow when different repos have different rules so I will not take that into account this time. 

In this PR you have worked on two separate issues and since the search functionality is already assigned to someone else I request you to remove those parts of the changes you made and leave only the navigation styling changes for now. Please feel free to raise an issue for the navigation styling and I will assign it to you.

**Comment by DIWAKARKASHYAP** [comment]
i am sorry and thankyou for your time and effort of reviewing

**Comment by DIWAKARKASHYAP** [comment]
@EmmaDawsonDev please check

File: components/Nav/NavPrimary.module.css [hunk 6]
```diff
@@ -19,26 +19,30 @@
...
   list-style: none;
+  margin: 2px;
+  margin-left: 10px;
 }
 
 .navListItem a {
   text-decoration: none;
-  padding: 16px;
-  display: block;
+  padding: 4px;
```
Comment (EmmaDawsonDev) [inline components/Nav/NavPrimary.module.css:28]: This padding feels too tight, I think maybe 8px would be more appropriate.

File: components/Nav/NavPrimary.module.css [hunk 7]
```diff
@@ -19,26 +19,30 @@
...
 
 .navListItem a:hover {
   background-color: var(--primaryLt);
 }
 
 .navList a:focus {
-  background-color: var(--primaryLt);
-  font-weight: bold;
+  background-color: var(--primary);
+  color: var(--white);
```
Comment (EmmaDawsonDev) [inline components/Nav/NavPrimary.module.css:39]: Setting the text to white means the color contrast on dark mode doesn't work. Please use the --text variable instead. Please also switch the background color back to primaryLt

**Review by EmmaDawsonDev — CHANGES_REQUESTED** [review]
Thanks for removing the stuff related to search. There's just a few more adjustments to make before this is ready to be merged. In the globals.css file there is a class called active. The styles there need to be updated to match the new design too.

**Comment by EmmaDawsonDev** [comment]
Hi @DIWAKARKASHYAP I'm not sure what's going on here or why you've deleted the primaryNav file. I am just waiting on you to adjust the .active class in the global.css file, as mentioned in one of my earlier comments. Let me know if you need any help.

**Comment by DIWAKARKASHYAP** [comment]
@EmmaDawsonDev apology i think if i delete then only changes gonna delete 
how can i restore this branch to previous commit , is this possible ?

**Comment by EmmaDawsonDev** [comment]
No problem, I wanted to check as I thought it was probably a mistake. You can use `git revert #commit` where #commit is the string of numbers and letters given to the commit. You can find it here on GitHub or by typing git log in the terminal

**Comment by DIWAKARKASHYAP** [comment]
@EmmaDawsonDev  thankyou soo much for help me ,
now i think its working perfectly

**Review by EmmaDawsonDev — APPROVED** [review]
Thanks for your help @DIWAKARKASHYAP and here's an article about good commit messages 😉 [Commit Messages](https://blog.ossph.org/how-to-write-a-good-git-commit-message/#:~:text=To%20set%20up%20a%20Git,t%20meet%20the%20specified%20format.)
